imagetest/fixtures.go

```diff
@@ -228,13 +235,19 @@ func (t *TestVM) SetCustomNetwork(network *Network, subnetwork *Subnetwork) erro
 	networkInterface := compute.NetworkInterface{
 		Network:    network.name,
 		Subnetwork: subnetworkName,
+		NetworkIP:  networkIP,
```

Comment:
now networkIP is required? what happens if someone doesn't want to specify it? i think we can pull that bit out to its own function, or have some different signatures for this one

Reply:
Just like subnetwork. "" empty means not setting. but we could provide a SetCustomNetworkWithNetworkIP dedicate for this and make SetCustomNetwork call  SetCustomNetworkWithNetworkIP with empty string without need to change other setup.